layouts/full_map/items.json

```diff
@@ -162,142 +162,5 @@
 				"key": "open_flags"
 			}
 		]
-	},
```

Comment:
this file shouldn't have anything to do with poptracker and should only ever be loaded by emotracker